nmsg/kafkaio.c

```diff
@@ -741,9 +794,14 @@ kafka_read_start(kafka_ctx_t ctx, uint8_t **buf, size_t *len)
 
 	if (ctx->message != NULL) {
 		if (ctx->message->err == RD_KAFKA_RESP_ERR_NO_ERROR) {
+			if (!ctx->connected)
+				_kafka_set_connection_state(ctx, __func__, true);
+
 			*buf = ctx->message->payload;
 			*len = ctx->message->len;
 			ctx->consumed++;
+			if ((ctx->consumed % 10000) == 0)
+				_nmsg_dprintf(4, "Consumed %ld\n",ctx->consumed);
```

Comment:
```
_nmsg_dprintf(4, "(%p) Delivered %ld messages\n", ctx, ctx->delivered);
```

Reply:
Fixed

Reply:
Note these two lines:

```
_nmsg_dprintf(6, "%s: Delivered %ld\n",__func__, ctx->delivered);
_nmsg_dprintf(6, "%s: Consumed %ld\n",__func__, ctx->consumed);
```

1. There should be a space before `__func__`
2. Delivered/consumed what? I added "messages" in suggested fix to make the output more meaningful...

Reply:
OK I see #1 fixed but not #2

Reply:
Missed message format change. Fixed